File: MyApplication/app/src/main/java/com/example/myapplication/Activity5.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class Activity5 extends MainActivity {
    @Override
    public void getNextButton() {
        Intent i = new Intent(Activity5.this, Activity2.class );
        startActivity(i);
    }

    @Override
    public void getPreviousButton() {
        Intent i = new Intent(Activity5.this, Activity4.class );
        startActivity(i);
    }

    @Override
    public String getAcitivityName() {
        return "Activity 5";
    }
}

File: MyApplication7/app/src/main/java/com/example/myapplication/Activity5.kt
package com.example.myapplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_5.*
import kotlinx.android.synthetic.main.activity_main.*

class Activity5 : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_5)

        button6.setOnClickListener {
            startActivity(Intent(this, Activity2::class.java))
        }

        button7.setOnClickListener {
            startActivity(Intent(this, Activity4::class.java))
        }

        supportActionBar!!.title = "Back"
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
    }
    override fun onSupportNavigateUp(): Boolean {
        onBackPressed()
        return true
    }
}
